### PR TITLE
Update librecad to 2.1.3 and use safe sources

### PR DESCRIPTION
**Pull request by scottmc**

### CODE REVIEW

File: media-gfx/librecad/librecad-2.1.3.recipe [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	"
+
+BUILD_REQUIRES="
+	libqt4${secondaryArchSuffix}_devel >= 4.8.0
+	boost${secondaryArchSuffix}_devel
+	devel:libfreetype$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:qmake$secondaryArchSuffix
```
Comment (korli) [inline media-gfx/librecad/librecad-2.1.3.recipe]: cmd:qmake$secondaryArchSuffix >= 4

File: media-gfx/librecad/librecad-2.1.3.recipe [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+LibreCAD has been re-structured, ported to new Qt versions and works natively \
+cross platform between OS X, Windows, Linux and Haiku."
+HOMEPAGE="http://librecad.org"
+COPYRIGHT="2015 LibreCAD"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/LibreCAD/LibreCAD/archive/2.1.3.tar.gz"
+CHECKSUM_SHA256="74c4ede409b13d0365c65c0cd52dba04f1049530f6df706dc905443d5e60db06"
+PATCHES="librecad_x86-2.1.3.patchset"
```
Comment (korli) [inline media-gfx/librecad/librecad-2.1.3.recipe]: you should consider renaming this patchset, librecad-$portVersion.patchset

**Comment by korli** [comment]
It would also be cool to try to build the master branch as it's Qt5 based. It might also be easier to upstream any needed change for Haiku.

**Comment by scottmc** [comment]
Main reason I picked this one to work on was to  fix the safe sources issue.  Didn't realize it was going to take hours to compile.

**Comment by scottmc** [comment]
tried cmd:qmake$secondaryArchSuffix >= 4
but haikuporter wan't able to find it.

**Comment by scottmc** [comment]
I only ran it one time and it seemed to work ok, that is until I tried to close the program and then it crashed.  Might have been a one time thing or it might crash every time you close it.  Need to do more testing.

**Pull request merged**